.travis.yml

```diff
@@ -1,34 +1,38 @@
 language: go
+sudo: required
```

Comment:
This would make it run on a dedicated VM, which i dont believe is needed for such a small project. Can we remove?

Reply:
Fair point - copy and paste :D Removed!